src/workflows/context/state_store.py

```diff
@@ -245,6 +245,14 @@ async def set(self, path: str, value: Any) -> None:
             # Assign the final value
             self._assign_step(current, segments[-1], value)
 
+    async def clear(self) -> None:
+        """Clear the state."""
+        async with self._lock:
+            try:
+                self._state = self._state.__class__()
```

Comment:
Not sure if viable, but we could make a copy of the intial state in the init, or the first time the state changes, and restore it in this method. We could even change the name to reset() or restore() to better convey what happens here. This way we wouldn't need to enforce default values for all params.